### PR TITLE
Gripper mujoco

### PR DESCRIPTION
**Pull request by yusukeurakami**
Hi, this is Yusuke Urakami with Panasonic USA R&D. Since your Blue Mujoco MJCF file doesn't have the working gripper, I added the functional gripper and also re-wrote the Gym environment file. Please read README file to know what I've changed.

### CODE REVIEW

**Comment by brentyi** [comment]
This looks really great, thanks @yusukeurakami!

Minor nitpick -- can we move `random_policy` out of the Mujoco environment?

File: koko_gym/envs/koko_reacher.py [hunk 1]
```diff
@@ -34,11 +59,29 @@ def reset_model(self):
...
-        return np.concatenate([
-            np.cos(theta),
-            np.sin(theta),
-            self.model.data.qpos.flat[2:],
-            self.model.data.qvel.flat[:2],
-            self.get_body_com("fingertip") - self.get_body_com("target")
-        ])
+        return np.concatenate([self.sim.data.qpos,self.sim.data.qvel,self.get_body_com("robotleftfingertip") - self.get_body_com("target")])
+
+    def random_policy(self):
```
Comment (brentyi) [inline koko_gym/envs/koko_reacher.py]: Minor nitpick -- can we move this policy outside of the environment?

**Review by brentyi — APPROVED** [review]
This looks really great, thanks @yusukeurakami!

File: koko_gym/envs/koko_reacher.py (hunk 1 above)
Comment (yusukeurakami) [reply koko_gym/envs/koko_reacher.py]: No problem with deleting the random policy from the environment. Yeah, that should not be the environment function.

Comment (yusukeurakami) [reply koko_gym/envs/koko_reacher.py]: Will you change it on your side?

Comment (brentyi) [reply koko_gym/envs/koko_reacher.py]: Yep, I can do it & merge

Comment (yusukeurakami) [reply koko_gym/envs/koko_reacher.py]: Okay! Great, then I will resolve this conversation.

**Pull request merged**